taylor-fatima.js

```diff
@@ -0,0 +1,143 @@
+
+//Part 1
+//#1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(num) {
+    console.log(Math.pow(num, 3))
+  }
+  calculateCube(2)
```

Comment:
careful with you indentation here! the closing bracket in line 8 should be equally as far left as the line where its corresponding opening bracket is, in line 6. And since line 9 is not inside the function block (aka, inside the curly brackets), it should also be as far left as line 6.